foreman/data_refinery_foreman/foreman/main.py

```diff
@@ -425,6 +425,20 @@ def requeue_survey_job(last_job: SurveyJob) -> None:
     The new survey job will have num_retries one greater than
     last_job.num_retries.
     """
+    nomad_host = get_env_variable("NOMAD_HOST")
+    nomad_port = get_env_variable("NOMAD_PORT", "4646")
+    nomad_client = Nomad(nomad_host, port=int(nomad_port), timeout=30)
+    # Maximum number of total jobs running at a time.
+    # We do this now rather than import time for testing purposes.
+    MAX_TOTAL_JOBS = int(get_env_variable_gracefully("MAX_TOTAL_JOBS", 1000))
+
+    lost_jobs = []
+    all_jobs = nomad_client.jobs.get_jobs()
```

Comment:
This could be called in `handle_survey_jobs` and then the length of it could be used to determine how many times to call this function, rather than calling for each individual job.